tokenserver/assignment/sqlnode/sql.py

```diff
@@ -224,6 +234,7 @@ def __init__(self, sqluri, create_tables=False, pool_size=100,
         self.services = get_cls('services', _Base)
         self.nodes = get_cls('nodes', _Base)
         self.users = get_cls('users', _Base)
+        self.settings = kw or {}
```

Comment:
`self.settings` seems unused in this file.

Reply:
Ah, good point.